template_feed/Microsoft.DotNet.Test.ProjectTemplates.6.0/content/MSTest-CSharp/.template.config/template.json

```diff
@@ -36,6 +36,19 @@
       "replaces": "net6.0",
       "defaultValue": "net6.0"
     },
+    "": {
+      "type": "parameter",
+      "datatype": "text",
+      "description": "Sets the LangVersion property in the created project file",
+      "defaultValue": "",
+      "replaces": "$(ProjectLanguageVersion)"
+    },
+    "nullable": {
```

Comment:
Was there a specific reason you added this and langVersion parameters? We decided to wait for feedback before doing it for the other templates.

Reply:
Ah thanks. Removing.